pkgs/development/python-modules/great-tables/default.nix

```diff
@@ -28,8 +33,6 @@ buildPythonPackage rec {
   version = "0.11.0";
   pyproject = true;
 
-  disabled = pythonOlder "3.9";
```

Comment:
why do you remove these? We still carry older python an pypy versions even though hydra doesn't build them anymore

Reply:
Other maintainers argued that keeping the `disabled` metadata when it is older than the python version we officially support was not that useful.
I also find it annoying to always check upstream information to update them. So, I have been removing them lately.

Reply:
Personally i prefer keeping them around, as they can save you a lot of time if you suddenly decide to build `python38Packages` for some reason. Perhaps some tooling to test these would be called for?

Reply:
(not blocking)